backend/src/main/resources/liquibase/create-versioning-tables-changelog.xml

```diff
@@ -235,4 +235,15 @@
         </createTable>
     </changeSet>
 
+    <changeSet id="add-label-index-in-labels-mapping" author="anandJ">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="labels_mapping" columnName="label_index"/>
+            </not>
+        </preConditions>
+        <addColumn tableName="labels_mapping">
+            <column name="label_index" type="int4" defaultValue="0"/>
+        </addColumn>
+    </changeSet>
+
```

Comment:
please revert, lets sort by label to get a consistent ordering

Reply:
Removed and modify all tag relevant test cases by alphanumeric order